pyriemann/transfer/_estimators.py

```diff
@@ -592,43 +601,474 @@ def fit_transform(self, X, y_enc, sample_weight=None):
+    """Center vectors in tangent space for transfer learning.
+
+    Center the vectors from each domain to zero on tangent space.
+
+    Parameters
+    ----------
+    target_domain : string
+        Name of the target domain in extended labels.
+
+    Attributes
+    ----------
+    centers_ : dict
+        Dictionary with key=domain_name and value=domain_mean_vector.
+
+    Notes
+    -----
+    .. versionadded:: 0.8
+    """
+
+    def __init__(self, target_domain):
+        """Init"""
+        self.target_domain = target_domain
+
+    def fit(self, X, y_enc):
+        """Fit TlTsCenter.
+
+        For each domain, calculates the mean of vectors of this domain.
+
+        Parameters
+        ----------
+        X : ndarray, shape (n_vectors, n_ts)
+            Set of tangent vectors.
+        y_enc : ndarray, shape (n_vectors,)
+            Extended labels for each vector.
+
+        Returns
+        -------
+        self : TlTsCenter instance
+            The TlTsCenter instance.
+        """
+        X, y, domains = decode_domains(X, y_enc)
+
+        self.centers_ = {}
+        for d in np.unique(domains):
+            idx = domains == d
+            self.centers_[d] = np.mean(X[idx], axis=0)
+
+        return self
+
+    def transform(self, X):
+        """Center vectors in the target domain.
+
+        .. note::
+           This method is designed for using at test time,
+           recentering all vectors in target domain.
+
+        Parameters
+        ----------
+        X : ndarray, shape (n_vectors, n_ts)
+            Set of tangent vectors.
+
+        Returns
+        -------
+        X_new : ndarray, shape (n_vectors, n_ts)
+            Set of centered tangent vectors.
+        """
+        X_new = X - self.centers_[self.target_domain]
+        return X_new
+
+    def fit_transform(self, X, y_enc):
+        """Fit TlTsCenter and then transform vectors.
+
+        For each domain, calculates the mean of vectors of this domain and
+        then recenters them to zero.
+
+        .. note::
+           This method is designed for using at training time. The output for
+           .fit_transform() will be different than using .fit() and
+           .transform() separately.
+
+        Parameters
+        ----------
+        X : ndarray, shape (n_vectors, n_ts)
+            Set of tangent vectors.
+        y_enc : ndarray, shape (n_vectors,)
+            Extended labels for each vector.
+
+        Returns
+        -------
+        X_new : ndarray, shape (n_vectors, n_ts)
+            Set of centered tangent vectors.
+        """
+        self.fit(X, y_enc)
+        X, y, domains = decode_domains(X, y_enc)
+
+        X_new = np.zeros_like(X)
+        for d in np.unique(domains):
+            idx = domains == d
+            X_new[idx] = X[idx] - self.centers_[d]
+        return X_new
+
+
+class TlTsNormalize(BaseEstimator, TransformerMixin):
+    """Normalize vectors in tangent space for transfer learning.
```

Comment:
Give more precision on what you mean by normalizing. Maybe say z-score ? Or explicitly say that you put mean zero and std to one.